### PR TITLE
Minor tests for utils and Exception

### PR DESCRIPTION
**Pull request by massimilianobraglia**
I have added some minor tests in `jymfony-exception` and `jymfony-util`. Please, review the exception part, because my environment had some test execution problems.

### CODE REVIEW

File: src/util/lib/String/escapeshellarg.js [hunk 1]
```diff
@@ -5,12 +5,12 @@ global.__jymfony = global.__jymfony || {};
 /*
  * Adapted from locutus.io
  */
-global.__jymfony.escapeshellarg = function escapeshellarg (arg) {
```
Comment (alekitto) [inline src/util/lib/String/escapeshellarg.js:8]: Please let global functions to have its own name and `this`

File: src/util/lib/String/escapeshellarg.js [hunk 2]
```diff
@@ -5,12 +5,12 @@ global.__jymfony = global.__jymfony || {};
...
+global.__jymfony.escapeshellarg = (arg) => {
     //  discuss at: http://locutus.io/php/escapeshellarg/
     // original by: Felix Geisendoerfer (http://www.debuggable.com/felix)
     // improved by: Brett Zamir (http://brett-zamir.me)
     //   example 1: escapeshellarg("kevin's birthday")
     //   returns 1: "'kevin\\'s birthday'"
 
-    return "'" + arg.replace(/[^\\]'/g, m => m.slice(0, 1) + '\\\'') + "'";
+    return `'${arg.replace(/[^\\]'/g, m => m.slice(0, 1) + '\\\'')}'`;
```
Comment (alekitto) [inline src/util/lib/String/escapeshellarg.js]: In this particular case I find the previous version clearer.
In general i prefer string interpolation of simple expressions (a variable or a property access, also via a getter) and concatenation for more complex expressions like this.

File: src/util/test/String/LevenshteinTest.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 require('../../lib/String/levenshtein');
 const expect = require('chai').expect;
 
-describe('Levenshtein distance', function () {
+describe('Levenshtein distance', () => {
```
Comment (alekitto) [inline src/util/test/String/LevenshteinTest.js]: Please do not use arrow functions in the `describe` function call. The reason is explained here: https://mochajs.org/#arrow-functions

File: src/util/test/String/TrimTest.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
+require('../../lib/String/ucwords');
```
Comment (alekitto) [inline src/util/test/String/TrimTest.js]: Did you mean: `../../lib/String/trim`?

File: src/util/test/String/UcFirstTest.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+require('../../lib/String/ucwords');
```
Comment (alekitto) [inline src/util/test/String/UcFirstTest.js]: Same here: wrong file requested

**Review by alekitto — CHANGES_REQUESTED** [review]

File: src/util/lib/String/escapeshellarg.js (hunk 1 above)
Comment (massimilianobraglia) [reply src/util/lib/String/escapeshellarg.js:8]: oh, ok! Didn't thought about that point

File: src/util/test/String/LevenshteinTest.js (hunk 3 above)
Comment (massimilianobraglia) [inline src/util/test/String/LevenshteinTest.js]: +1

File: src/util/lib/String/escapeshellarg.js (hunk 2 above)
Comment (massimilianobraglia) [reply src/util/lib/String/escapeshellarg.js]: I understand your point of view, but this case was clear enough (at least for me).
Anyway, no problem, reverting

Comment (alekitto) [reply src/util/lib/String/escapeshellarg.js]: Thinkin' about it: we should define it better and write down a code style rules documentation file

Comment (massimilianobraglia) [reply src/util/lib/String/escapeshellarg.js]: You know me: it is fundamental to have a CS rule file :)

**Review by alekitto — APPROVED** [review]

**Pull request merged**